web/atlas.json

```diff
@@ -6673,7 +6672,7 @@
 {"id": "txnbqc", "submitted_by": "DXPower", "name": "DotA2 Health/Mana Bar", "description": "The in-game health/mana bar aka HP Bar in the 5v5 hero strategy game DotA 2.\n\nThe green represents current health, and the blue represents current mana. The player dies when health is 0, and cannot cast spells when mana is 0.\n\nThe current health is 322 in this depiction.", "website": "https://www.dota2.com/home", "subreddit": "/r/DotA2", "center": [14.5, 140.5], "path": [[0.5, 137.5], [27.5, 137.5], [27.5, 143.5], [0.5, 143.5], [0.5, 137.5], [0.5, 137.5]]},
 {"id": "txnbfh", "submitted_by": "ferreju", "name": "\ud0dc\uadf9\uae30 Taegeukgi of South Korea", "description": "The National Flag : \ud0dc\uadf9\uae30 [Taegeukgi]", "website": "https://www.mois.go.kr/eng/sub/a03/nationalSymbol/screen.do", "subreddit": "/r/southkorea", "center": [1221.5, 687.5], "path": [[1200.5, 674.5], [1200.5, 700.5], [1241.5, 700.5], [1241.5, 674.5]]},
-{"id": "txnb2h", "submitted_by": "Lord-ofDerp", "name": "VoidPunk flag", "description": "the flag of the Void Punk community. After being destroyed several times, the flag survived because of an alliance between TierZoo, Binding of Issac, Inscyrption, and Cascadia. ", "website": "", "subreddit": "/r/voidpunk", "center": [55.5, 356.5], "path": [[48.5, 350.5], [61.5, 350.5], [61.5, 360.5], [61.5, 361.5], [48.5, 361.5], [48.5, 350.5]]},
+{"id": "txnb2h", "submitted_by": "chocomint1", "name": "VoidPunk Flag", "description": "The flag of the Void Punk community. \nVoidpunk is a subculture for those who often feel rejected or disconnected from humanity. This is popular within aro-spec and ace-spec communities but also includes groups such as neurodivergent people, disabled people, non-binary people and other groups that are frequently dehumanized or called 'subhuman' by society. Voidpunk is based on embracing the idea of being 'not human', and recognizing the bigotry by taunting their oppressors.\n\nThis particular instance of the flag was destroyed several times together with surrounding groups, and a slightly larger flag managed to stay in the end with the help of neighboring communities such as TierZoo, The Binding of Issac, Inscryption, and Cascadia.", "website": "", "subreddit": "/r/voidpunk", "center": [55.5, 356.5], "path": [[48.5, 350.5], [61.5, 350.5], [61.5, 360.5], [61.5, 361.5], [48.5, 361.5], [48.5, 350.5]]},

```

Comment:
Flag is not a proper noun, so leaving it uncapitalized is correct.

Reply:
While I agree that it's not a proper noun, it's part of a title and so everything except articles and conjunctions get capitalized ;p

Reply:
I treat these more as labels than titles. So those would use standard casing, not title case.